cmd/pebble/cmd_run.go

```diff
@@ -180,8 +239,13 @@ out:
 	for {
 		select {
 		case sig := <-ch:
-			logger.Noticef("Exiting on %s signal.\n", sig)
-			break out
+			for _, shutdownSig := range shutdownSignals {
+				if shutdownSig.String() == sig.String() {
+					logger.Noticef("Exiting on %s signal.\n", sig)
+					break out
+				}
+			}
+			logger.Noticef("Signal %s is not a shut down signal. Ignoring.\n", sig)
```

Comment:
I'm not sure this should be logged, as they didn't ask for it. Or maybe `logger.Debugf`.